bot/slash_commands/update.rb

```diff
@@ -0,0 +1,31 @@
+SlackRubyBotServer::Events.configure do |config|
+  config.on :command, '/update' do |command|
+    channel_name = command[:channel_name]
+    channel_id = command[:channel_id]
+    command.logger.info "Updating the incident in #{channel_name}"
+
+    slack_client = Slack::Web::Client.new(token: ENV['SLACK_TOKEN'])
+
+    begin
+      if channel_name.include? 'incident'
+        slack_client.views_open(trigger_id: command[:trigger_id], view: update_payload)
+        Rails.cache.write('channel', channel_id)
+        { text: 'You’ve updated the incident.' }
+      else
+        { text: 'This is not an incident channel.' }
+      end
+    rescue Slack::Web::Api::Errors::NotInChannel
+      { text: 'This is not an incident channel.' }
+    rescue Slack::Web::Api::Errors::CantInvite
+      { text: 'Cant invite this user' }
+    rescue Slack::Web::Api::Errors::NoUser
```

Comment:
Good to cover these. Should we also be handling errors like this for the incident-create action?

Reply:
Agree – but not too sure on the best errors to cover. We did have an incident recently where the incident description was over the character count.. it errored silently though and still proceeded to create the incident and perform the rest of the actions

Reply:
We do cover some in `bot/actions/slack_incident_listener.rb`

Reply:
have added an extra one